README.md

```diff
@@ -1,2 +1,8 @@
 # LauraTest
 learning document
+
+This is my first sentence on github
```

Comment:
You forgot a period at the end of this sentence.

Reply:
OK, i will fix it.